terraform/main.tf

```diff
@@ -396,3 +396,31 @@ resource "google_service_account_iam_member" "gce-default-account-iam" {
   role               = "roles/iam.serviceAccountUser"
   member             = "serviceAccount:345950277072@cloudbuild.gserviceaccount.com"
 }
+
+resource "google_secret_manager_secret" "pudl_usage_metrics_db_connection_string" {
+  secret_id = "pudl-usage-metrics-db-connection-string"
+  replication {
+    auto {}
+  }
+}
```

Comment:
I figured I'd save the connection string in case we need to reconnect the db to superset.